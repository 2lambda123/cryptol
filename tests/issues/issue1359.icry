:set prover=sbv-z3
:safe \a -> sortBy (\c1 c2 -> c2 != ' ') (split`{8} a)
:safe \(a : [64]) (i : Integer) -> (split`{8} a)@(max 0 (min i 7))

:set prover=w4-z3
:safe \a -> sortBy (\c1 c2 -> c2 != ' ') (split`{8} a)
:safe \(a : [64]) (i : Integer) -> (split`{8} a)@(max 0 (min i 7))
