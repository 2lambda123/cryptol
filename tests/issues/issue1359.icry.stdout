Loading module Cryptol
Safe
Safe
Safe
Safe
